CHANGELOG.md

```diff
@@ -102,6 +102,14 @@ If you're importing JavaScript modules individually, you should check any refere
 
 This change was introduced in [pull request #2426: Rename exported JavaScript modules to include component name](https://github.com/alphagov/govuk-frontend/pull/2426).
 
+#### Update the HTML for error messages
+
+We’ve updated the HTML for the error message component to use a paragraph element instead of a span element.
```

Comment:
Do we need to say why we've done this? The PR says because it's more semantically correct. Will users know what we mean by that, or do we need to specify how this change advantages them?